pinentry-mac fails to build (doesn't finish 'Fill Info.plist')

## Issue description
This is the build output I get (it's pulled in as part of a package I set up for my environment using buildEnv)

         ~/C/P/devnix   playground-darwin +  nix-env -i pxc-common-cli-env                                                                                                                                         34.5s  Thu Jun 22 13:17:40 2017
        replacing old ‘pxc-common-cli-env’
        installing ‘pxc-common-cli-env’
        these derivations will be built:
          /nix/store/0iyq9a9cbhnzp7c70ap82jmwxrq7piqx-vimplugin-vim-airline-themes-2017-05-24.drv
          /nix/store/1ik7p1ciq7wjm3wj9cd9yic1n7bb4i8b-vimplugin-fugitive-2017-06-08.drv
          /nix/store/4a1gq6qw1djc357s5dkfrxsgfc7aa0xh-vimplugin-vim-colorschemes-2016-08-01.drv
          /nix/store/6lnhi14cpvnmz7l38x6py2n0vi73rk72-vimplugin-Supertab-2016-11-27.drv
          /nix/store/6yh1z84xiqypyynq6finv6pcmzyjipgp-vimplugin-ctrlp-vim-2017-06-04.drv
          /nix/store/bnx7vcfdwj4fad5ln5dj519v0zzf6c1p-vimplugin-UltiSnips-2017-01-20.drv
          /nix/store/kjj069xp8ngkn6mhz2j29407pwd0z6iz-vimplugin-vim-nix-2017-04-30.drv
          /nix/store/bxpf74h2pvq68g5v3vk404a7rlgvnnyw-vim-pack-dir.drv
          /nix/store/h23yq372y2ka18m8r7y4p6pq6ycn88cf-vimplugin-sensible-2017-05-09.drv
          /nix/store/hrxd1mj5vvbgbabfa2xzvf09v6q0h6rc-vimplugin-vim-tmux-navigator-2017-06-01.drv
          /nix/store/ichvdnhcpdii84828bd6jhnhz5z5y6rz-vimplugin-vim-airline-2017-05-08.drv
          /nix/store/icvxf769pdcaxba93wk3dsrcspcxifda-vimplugin-vim2nix.drv
          /nix/store/iv3pc1wf8a0220pv8iz6dvh6ad7zjw4g-vimplugin-Syntastic-2017-05-30.drv
          /nix/store/jij3cnn6ix6f9yq994sadh6w1gbs0h1v-vimplugin-Tabular-2016-05-04.drv
          /nix/store/mprjn60lzz67crdmly6s3wys4icpan0v-vimplugin-VimOutliner-2017-05-03.drv
          /nix/store/qqih87icflb0m00s41z5qvn4gr4i3588-vimplugin-vim-scala-2017-04-29.drv
          /nix/store/r9nw4kp9516shywiywfxwr19sbmwgc5s-vimplugin-vim-gitgutter-2017-06-08.drv
          /nix/store/32kkdlr2zxprw23pm6x4wpl3k372y83m-vimrc.drv
          /nix/store/6mby8ar41s5lkvmh3s2bmfbv3v95f2s0-ranger-1.9.0b5.drv
          /nix/store/m42sgiap42z2159aqb2068vrvvh6s4p5-pinentry-mac-0.9.4.drv
          /nix/store/6p9laqwprw9yh9y7104iqkq1jp8kzpb3-gnupg-2.0.30.drv
          /nix/store/dbswqr60j9xjpf9i2qkjbdpng3nrzvbf-weechat-1.8.drv
          /nix/store/dncx6xqsbbqq9zai4qjsvcrasba0i9fm-vim.drv
          /nix/store/sr9f099czll25mb51lkzbnhl6x3s9dl8-gnupg-2.1.21.drv
          /nix/store/v92rqlk7rgy3njx6k5c99sn05gcr1irc-keychain-2.8.3.drv
          /nix/store/zj4k4zq2f6n72zd3wki71p4b4plp48gi-password-store-1.7.1.drv
          /nix/store/pr1bn8lcf1y9038zgyrxj4qri869vlq8-pxc-common-cli-env.drv
        these paths will be fetched (2.97 MiB download, 35.82 MiB unpacked):
          /nix/store/5blbx1lggf76pacnp1kdv1m66k3yrvmi-super-user-spark-0.3.2.0-dev
          /nix/store/5xdqpxzda6dkbs5rsj78spcfaivd757z-scientific-0.3.4.15
          /nix/store/6bs5zp10ghkcsbshb3r3rglf6phfyx5v-aeson-pretty-0.8.4
          /nix/store/dl5j61fn6pmqdv79f0rn0r0ll2hmj8ik-attoparsec-0.13.1.0
          /nix/store/kvl98m9rcqbbp8c8ayd9zq0ylgc59466-aeson-1.0.2.1
        fetching path ‘/nix/store/5xdqpxzda6dkbs5rsj78spcfaivd757z-scientific-0.3.4.15’...

        *** Downloading ‘https://cache.nixos.org/nar/0s4l3brpqy1qvpjldm153alrxfdyq867npp1psx40ybwaammkwbf.nar.xz’ (signed by ‘cache.nixos.org-1’) to ‘/nix/store/5xdqpxzda6dkbs5rsj78spcfaivd757z-scientific-0.3.4.15’...
          % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                         Dload  Upload   Total   Spent    Left  Speed
        100  152k  100  152k    0     0   524k      0 --:--:-- --:--:-- --:--:--  675k

        building path(s) ‘/nix/store/vbqm65l50rcbcjancsbqffjjdrr5sxnx-pinentry-mac-0.9.4’
        unpacking sources
        unpacking source archive /nix/store/8rpgx1lxaxcvf6xsyajpjrz38yk69g9r-pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src
        source root is pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src
        patching sources
        configuring
        no configure script, doing nothing
        building
        make flags: SHELL=/nix/store/w37k77yaagi7kdyiy263cma1bnrxz0ry-bash-4.4-p12/bin/bash
        xcodebuild -project pinentry-mac.xcodeproj -target pinentry-mac -configuration Release build
        clang-4.0: warning: using sysroot for 'k' but targeting 'MacOSX' [-Wincompatible-sysroot]
        /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src/Source/pinentry-current/assuan/assuan-buffer.c:80:18: warning: implicit conversion loses integer precision: 'ssize_t' (aka 'long') to 'int' [-Wshorten-64-to-32]
              nwritten = write (fd, buffer, length);
                       ~ ^~~~~~~~~~~~~~~~~~~~~~~~~~
        /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src/Source/pinentry-current/assuan/assuan-buffer.c:117:11: warning: implicit conversion loses integer precision: 'ssize_t' (aka 'long') to 'int' [-Wshorten-64-to-32]
              n = read (fd, buf, nleft);
                ~ ^~~~~~~~~~~~~~~~~~~~~
        /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src/Source/pinentry-current/assuan/assuan-buffer.c:382:32: warning: implicit conversion loses integer precision: 'size_t' (aka 'unsigned long') to 'int' [-Wshorten-64-to-32]
          ctx->outbound.data.linelen = linelen;
                                     ~ ^~~~~~~
        3 warnings generated.
        Set: Entry, "CommitHash", Does Not Exist
        Set: Entry, "BuildNumber", Does Not Exist
        Set: Entry, "CFBundleVersion", Does Not Exist
        Set: Entry, "CFBundleShortVersionString", Does Not Exist
        File Doesn't Exist, Will Create: ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/pinentry-mac.app/Contents/Info.plist
        Build settings from the command line:
            DERIVED_DATA_DIR=.
            SDKROOT=macosx10.10

        === BUILD TARGET pinentry-mac OF PROJECT pinentry-mac WITH CONFIGURATION Release ===

        Check dependencies

        Write auxiliary files
        /bin/mkdir -p ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/pinentry-mac.hmap
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/pinentry-mac-own-target-headers.hmap
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/pinentry-mac-all-target-headers.hmap
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/pinentry-mac-all-non-framework-target-headers.hmap
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/pinentry-mac-generated-files.hmap
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/pinentry-mac-project-headers.hmap
        /bin/mkdir -p ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Objects-normal/x86_64
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Objects-normal/x86_64/pinentry-mac.LinkFileList
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-30FC955F129818AE0039CEF8.sh
        chmod 0755 ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-30FC955F129818AE0039CEF8.sh
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-30185C6B1A98CA980025E008.sh
        chmod 0755 ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-30185C6B1A98CA980025E008.sh
        write-file ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-307377841AA4B7DA0097BCDE.sh
        chmod 0755 ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-307377841AA4B7DA0097BCDE.sh

        Create product structure

        Touch ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/pinentry-mac.app
            cd /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src
            /usr/bin/touch -c ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/pinentry-mac.app

        PBXCp ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/pinentry-mac.app/Contents/Resources/German.lproj /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src/Resources/German.lproj
            cd /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src
            export DEVELOPER_DIR=/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper
            export PATH=/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/Developer/usr/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/local/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/local/bin:/nix/store/1yxlcsbzwmf4y1kghjgqi7j28rlhx2y9-nixpkgs.xctoolchain/usr/bin:/nix/store/1yxlcsbzwmf4y1kghjgqi7j28rlhx2y9-nixpkgs.xctoolchain/usr/libexec:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/usr/bin:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/usr/local/bin:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/Tools:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/bin:/nix/store/80clgc15yj8p9y5kxinaph5glcgf95vv-libiconv-osx-10.11.6/bin:/nix/store/5k073yr9hsvr6zbm7dajkg64198jgyar-ncurses-5.9-dev/bin:/nix/store/mz1lcrajfibaramygjcqk2x27f7gdmvy-ncurses-5.9/bin:/nix/store/6v28y66hq8hm65cff4fx7z6kvwv0dcyw-clang-wrapper-4.0.0/bin:/nix/store/b70ipgiymp029ggj40hmywg6wswcgrx8-clang-4.0.0/bin:/nix/store/2zfsswnz483hjsdzwck2wjhc2xsisq3c-cctools-binutils-darwin/bin:/nix/store/6sq1ypmcx06pn4xk66kg5znrgmjiaw9k-coreutils-8.27/bin:/nix/store/0i9c75vp3n4ign9a9rf7ycsqhdjikmh2-findutils-4.6.0/bin:/nix/store/kky1fyl4syr8lvhmac3bx33gsbp01wvi-diffutils-3.6/bin:/nix/store/rxqgbx66dhz5n6m5pwqzaih7lkb1i7fj-gnused-4.4/bin:/nix/store/iijhiiy753cas31an7iin9v5cqrdizmk-gnugrep-3.0/bin:/nix/store/a8wbpbcvikwcmrghvgh810f6jv2px4gz-gawk-4.1.4/bin:/nix/store/6gas2fjnhqkjdf53qhyvxk96pi6zb5sq-gnutar-1.29/bin:/nix/store/2fvlsl03gqz67dxxzp5gs6j3l9jmbg6h-gzip-1.8/bin:/nix/store/m4kvas5dayj2asmparb4gy65gzk30cda-bzip2-1.0.6.0.1-bin/bin:/nix/store/a4ryd0g1gh03cg563mh6yays4jd9x4qp-gnumake-4.2.1/bin:/nix/store/w37k77yaagi7kdyiy263cma1bnrxz0ry-bash-4.4-p12/bin:/nix/store/qrx2yl2ykw8yqci28mji195m3i5p37m7-patch-2.7.5/bin:/nix/store/fbcmrgib009fd0008097w0yigp6z1kxw-xz-5.2.3-bin/bin
            builtin-copy -resolve-src-symlinks -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -strip-debug-symbols /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src/Resources/German.lproj ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/pinentry-mac.app/Contents/Resources

        CompileC ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Objects-normal/x86_64/assuan-buffer.o Source/pinentry-current/assuan/assuan-buffer.c normal x86_64 c com.apple.compilers.llvm.clang.1_0.compiler
            cd /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src
            export DEVELOPER_DIR=/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper
            export PATH=/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/Developer/usr/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/local/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/local/bin:/nix/store/1yxlcsbzwmf4y1kghjgqi7j28rlhx2y9-nixpkgs.xctoolchain/usr/bin:/nix/store/1yxlcsbzwmf4y1kghjgqi7j28rlhx2y9-nixpkgs.xctoolchain/usr/libexec:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/usr/bin:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/usr/local/bin:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/Tools:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/bin:/nix/store/80clgc15yj8p9y5kxinaph5glcgf95vv-libiconv-osx-10.11.6/bin:/nix/store/5k073yr9hsvr6zbm7dajkg64198jgyar-ncurses-5.9-dev/bin:/nix/store/mz1lcrajfibaramygjcqk2x27f7gdmvy-ncurses-5.9/bin:/nix/store/6v28y66hq8hm65cff4fx7z6kvwv0dcyw-clang-wrapper-4.0.0/bin:/nix/store/b70ipgiymp029ggj40hmywg6wswcgrx8-clang-4.0.0/bin:/nix/store/2zfsswnz483hjsdzwck2wjhc2xsisq3c-cctools-binutils-darwin/bin:/nix/store/6sq1ypmcx06pn4xk66kg5znrgmjiaw9k-coreutils-8.27/bin:/nix/store/0i9c75vp3n4ign9a9rf7ycsqhdjikmh2-findutils-4.6.0/bin:/nix/store/kky1fyl4syr8lvhmac3bx33gsbp01wvi-diffutils-3.6/bin:/nix/store/rxqgbx66dhz5n6m5pwqzaih7lkb1i7fj-gnused-4.4/bin:/nix/store/iijhiiy753cas31an7iin9v5cqrdizmk-gnugrep-3.0/bin:/nix/store/a8wbpbcvikwcmrghvgh810f6jv2px4gz-gawk-4.1.4/bin:/nix/store/6gas2fjnhqkjdf53qhyvxk96pi6zb5sq-gnutar-1.29/bin:/nix/store/2fvlsl03gqz67dxxzp5gs6j3l9jmbg6h-gzip-1.8/bin:/nix/store/m4kvas5dayj2asmparb4gy65gzk30cda-bzip2-1.0.6.0.1-bin/bin:/nix/store/a4ryd0g1gh03cg563mh6yays4jd9x4qp-gnumake-4.2.1/bin:/nix/store/w37k77yaagi7kdyiy263cma1bnrxz0ry-bash-4.4-p12/bin:/nix/store/qrx2yl2ykw8yqci28mji195m3i5p37m7-patch-2.7.5/bin:/nix/store/fbcmrgib009fd0008097w0yigp6z1kxw-xz-5.2.3-bin/bin
            /nix/store/1yxlcsbzwmf4y1kghjgqi7j28rlhx2y9-nixpkgs.xctoolchain/usr/bin/clang -x c -arch x86_64 -isysroot /nix/store/k075n3dcpiqf4zzrsi2qnxrxjdimdq59-nix.nixpkgs.sdk -mmacosx-version-min=10.10 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -DHAVE_CONFIG_H -DFALLBACK_CURSES -fasm-blocks -fno-common -fstrict-aliasing -fpascal-strings -Os -Wno-trigraphs -Wempty-body -Winfinite-recursion -Wunreachable-code -Wdeprecated-declarations -Wno-missing-field-initializers -Wno-newline-eof -Wno-missing-prototypes -Wpointer-sign -Wswitch -Wno-four-char-constants -Wno-conversion -Wbool-conversion -Wconstant-conversion -Wenum-conversion -Wint-conversion -Wno-sign-conversion -Wshorten-64-to-32 -Wno-missing-braces -Wparentheses -Wno-shadow -Wuninitialized -Wno-unknown-pragmas -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-value -Wunused-variable -I./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/pinentry-mac.hmap -I./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/include -I./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/DerivedSources/x86_64 -I./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/DerivedSources -F./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release -MMD -MT dependencies -MF ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Objects-normal/x86_64/assuan-buffer.d -c /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src/Source/pinentry-current/assuan/assuan-buffer.c -o ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Objects-normal/x86_64/assuan-buffer.o

        CopyPlistFile ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/pinentry-mac.app/Contents/Resources/Main.nib Resources/Main.nib
            cd /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src
            export DEVELOPER_DIR=/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper
            export PATH=/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/Developer/usr/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/local/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/bin:/nix/store/1jda6s4mykckzqgzy54hnblajngfbwxp-nixpkgs.platform/usr/local/bin:/nix/store/1yxlcsbzwmf4y1kghjgqi7j28rlhx2y9-nixpkgs.xctoolchain/usr/bin:/nix/store/1yxlcsbzwmf4y1kghjgqi7j28rlhx2y9-nixpkgs.xctoolchain/usr/libexec:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/usr/bin:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/usr/local/bin:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/Tools:/nix/store/ghfszjq2196xr2phc9x1sr2blrx2p92f-xcbuild-wrapper/bin:/nix/store/80clgc15yj8p9y5kxinaph5glcgf95vv-libiconv-osx-10.11.6/bin:/nix/store/5k073yr9hsvr6zbm7dajkg64198jgyar-ncurses-5.9-dev/bin:/nix/store/mz1lcrajfibaramygjcqk2x27f7gdmvy-ncurses-5.9/bin:/nix/store/6v28y66hq8hm65cff4fx7z6kvwv0dcyw-clang-wrapper-4.0.0/bin:/nix/store/b70ipgiymp029ggj40hmywg6wswcgrx8-clang-4.0.0/bin:/nix/store/2zfsswnz483hjsdzwck2wjhc2xsisq3c-cctools-binutils-darwin/bin:/nix/store/6sq1ypmcx06pn4xk66kg5znrgmjiaw9k-coreutils-8.27/bin:/nix/store/0i9c75vp3n4ign9a9rf7ycsqhdjikmh2-findutils-4.6.0/bin:/nix/store/kky1fyl4syr8lvhmac3bx33gsbp01wvi-diffutils-3.6/bin:/nix/store/rxqgbx66dhz5n6m5pwqzaih7lkb1i7fj-gnused-4.4/bin:/nix/store/iijhiiy753cas31an7iin9v5cqrdizmk-gnugrep-3.0/bin:/nix/store/a8wbpbcvikwcmrghvgh810f6jv2px4gz-gawk-4.1.4/bin:/nix/store/6gas2fjnhqkjdf53qhyvxk96pi6zb5sq-gnutar-1.29/bin:/nix/store/2fvlsl03gqz67dxxzp5gs6j3l9jmbg6h-gzip-1.8/bin:/nix/store/m4kvas5dayj2asmparb4gy65gzk30cda-bzip2-1.0.6.0.1-bin/bin:/nix/store/a4ryd0g1gh03cg563mh6yays4jd9x4qp-gnumake-4.2.1/bin:/nix/store/w37k77yaagi7kdyiy263cma1bnrxz0ry-bash-4.4-p12/bin:/nix/store/qrx2yl2ykw8yqci28mji195m3i5p37m7-patch-2.7.5/bin:/nix/store/fbcmrgib009fd0008097w0yigp6z1kxw-xz-5.2.3-bin/bin
            builtin-copyPlist --outdir ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Products/Release/pinentry-mac.app/Contents/Resources -- Resources/Main.nib

        PhaseScriptExecution 'Fill Info.plist' ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-30185C6B1A98CA980025E008.sh
            cd /private/var/folders/4_/_4vg56r13tq4dpb8064qmlvh0000gn/T/nix-build-pinentry-mac-0.9.4.drv-0/pinentry-mac-d60aa902644a1f0126ec50e79937423a3a7c3bc4-src
            /bin/sh -c ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-30185C6B1A98CA980025E008.sh

        ** BUILD FAILED **

        The following build commands failed:
            PhaseScriptExecution 'Fill Info.plist' ./pinentry-mac-bjnprpydovwtpvajwyvhdeyuoopc/Build/Intermediates/pinentry-mac.build/Release/pinentry-mac.build/Script-30185C6B1A98CA980025E008.sh
        (1 failure)
        make: *** [Makefile:9: build/Release/pinentry-mac.app/Contents/MacOS/pinentry-mac] Error 1
        builder for ‘/nix/store/m42sgiap42z2159aqb2068vrvvh6s4p5-pinentry-mac-0.9.4.drv’ failed with exit code 2
        fetching path ‘/nix/store/dl5j61fn6pmqdv79f0rn0r0ll2hmj8ik-attoparsec-0.13.1.0’...
        cannot build derivation ‘/nix/store/6p9laqwprw9yh9y7104iqkq1jp8kzpb3-gnupg-2.0.30.drv’: 1 dependencies couldn't be built
        cannot build derivation ‘/nix/store/sr9f099czll25mb51lkzbnhl6x3s9dl8-gnupg-2.1.21.drv’: 1 dependencies couldn't be built
        killing process 14284
        cannot build derivation ‘/nix/store/pr1bn8lcf1y9038zgyrxj4qri869vlq8-pxc-common-cli-env.drv’: 1 dependencies couldn't be built
        error: build of ‘/nix/store/pr1bn8lcf1y9038zgyrxj4qri869vlq8-pxc-common-cli-env.drv’ failed

Am I missing something?

### Steps to reproduce
Try to build pinentry-mac on Nixpkgs unstable


## Technical details

* System: macOS Sierra
* Nix version: nix-env (Nix) 1.11.9
* Nixpkgs version: "17.09pre-git"
* Sandboxing enabled: build-use-sandbox = false

--- a/pkgs/tools/security/pinentry-mac/default.nix
+++ b/pkgs/tools/security/pinentry-mac/default.nix
@@ -6,8 +6,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "matthewbauer";
     repo = "pinentry-mac";
-    rev = "d60aa902644a1f0126ec50e79937423a3a7c3bc4";
-    sha256 = "0xp4rdyj0mw6gg1z1wraggb1qlkjb5845mibrz3nj0l692da52nq";
+    rev = "6dfef256c8ea32d642fea847f27d800f024cf51e";
+    sha256 = "0g75302697gqcxyf2hyqzvcbd5pyss1bl2xvfd40wqav7dlyvj83";
   };
 
   buildInputs = [ xcbuild libiconv Cocoa ncurses ];

